hutch_python/load_conf.py

```diff
@@ -3,33 +3,33 @@
 The file's specification can be found on the `yaml_files` page.
 """
 import logging
-import yaml
 from copy import copy
 from pathlib import Path
 from socket import gethostname
 from types import SimpleNamespace
 
-from . import mpl_config  # noqa: F401 # isort: ignore
```

Comment:
isort is being mean...

Reply:
Try bumping its version in pre-commit to latest (5.6.4) 🤞

Reply:
No permutations seem to work. There is a difference between where isort moves this line with or without an ignore and with or without a skip, but none of the permutations keep it where it needs to be. I'm going to look into other options for configuring isort.

Reply:
Seems to be an isort bug then, I suppose. I'll make an issue to refactor this down the line so we can still use it.